src/cmds/config.js

```diff
@@ -96,31 +96,49 @@ exports.handler = async function handler(argv) {
     const db = client(server);
     let result;
     if (!options) {
-      result = await db.getServiceConfiguration(argv.mount, argv.minimal);
```

Comment:
It's a bit wasteful but reassembling the minimal response on the client makes the pretty printing a bit easier to deal with IMO.